pgsqltoolsservice/query/query.py

```diff
@@ -71,16 +71,19 @@ def __init__(self, owner_uri: str, query_text: str, query_execution_settings: Qu
             formatted_text = sqlparse.format(batch_text, strip_comments=True).strip()
             if not formatted_text or formatted_text == ';':
                 continue
+
+            sql_statement_text = batch_text
```

Comment:
You don't need this, we can directly use batch_text

Reply:
Actually we do, never mind. The changes looks good.

Reply:
batch_text is input data in a way, like a function argument, so I wanted to preserve it. But yes, I can change that.

Reply:
Yes, lets keep it the way it is. I missed that you were manipulating it.